app/controllers/project/index.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+const { Controller, computed, inject: { service } } = Ember;
+
+export default Controller.extend({
+  store: service(),
+  currentUser: service(),
+
+  user: computed.alias('currentUser.user'),
+
+  actions: {
+    saveSubscription(amount) {
+      let project = this.get('project');
+      let queryParams = { amount };
+      this.transitionToRoute('project.donate', project, { queryParams });
```

Comment:
Yeah, last parameter of `transitionToRoute` needs to be a hash with a `queryParams` property, which is also a hash.

http://emberjs.com/api/classes/Ember.Controller.html#method_transitionToRoute

It's at the end of that section